src/simulator_core/entities/assets/esdl_asset_object.py

```diff
@@ -39,13 +39,13 @@ def __init__(self, asset: esdl.Asset) -> None:
         """
         self.esdl_asset = asset
 
-    def get_property(self, esdl_property_name: str, default_value: Any) -> (Any, bool):
+    def get_property(self, esdl_property_name: str, default_value: Any) -> Tuple[Any, bool]:
         """Get property value from the esdl_asset based on the "ESDL" name.
 
         :return: Tuple with the value of the property and a boolean indicating whether the property
         was found in the esdl_asset.
         """
         try:
-            return (getattr(self.esdl_asset, esdl_property_name), True)
+            return getattr(self.esdl_asset, esdl_property_name), True
```

Comment:
Interessant dat hier geen opmerking wordt geplaatst dat de output geen Tuple is of komt dat doordat je hem aangeeft met Tuple[Any, bool]?

Reply:
Python maakt hier een tuple van, en de haakjes hoeven dan niet